packages/common/src/types/dbt.ts

```diff
@@ -290,6 +294,10 @@ export type DbtMetricLightdashMetadata = {
     groups?: string[];
     show_underlying_values?: string[];
     filters: Record<string, any>[];
+    default_time_dimension?: {
+        field: string;
+        interval: TimeFrames;
+    };
```

Comment:
this should not be required afaik

Reply:
exactly! ty, removed both metric-related changes

Reply:
https://github.com/lightdash/lightdash/pull/12505/commits/8b095cc40c7ef829e26f999d785cf98e81c860e2